### ISSUE TITLE
Add label encoder

### ISSUE THREAD

**Issue by onurtemizkan** [onurtemizkan/fleuret#6]
Related: #5.

Add a label encoder to encode labels into integer values.  See [scikit's implementation](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html#sklearn.preprocessing.LabelEncoder).
